fuzzers/074-dump_all/generate_grid.py

```diff
@@ -605,6 +606,16 @@ def main():
         tileconn, raw_node_data = generate_tileconn(
             pool, node_tree, nodes, wire_map, grid)
 
+        for data in tileconn:
+            data['wire_pairs'] = tuple(
+                sorted(
+                    data['wire_pairs'],
+                    key=lambda x: tuple(extract_numbers(s) for s in x)))
+
+        tileconn = tuple(
+            sorted(
+                tileconn, key=lambda x: (x['tile_types'], x['grid_deltas'])))
```

Comment:
Tile types doesn't tend to have many numbers, so I don't think it is required.  Also it would required fixing the `extract_numbers` formatting.  So maybe leave as follow on?

Reply:
Hrm, I think I was thinking about the `site_types` where I wanted the first site to be the first value....